src/Time/TimeSteppers/AdamsBashforthN.hpp

```diff
@@ -45,9 +45,52 @@ class History;
 
 namespace TimeSteppers {
 
-/// \ingroup TimeSteppersGroup
-///
-/// An Nth Adams-Bashforth time stepper.
+/*!
+ * \ingroup TimeSteppersGroup
+ *
+ * An Nth Adams-Bashforth time stepper.
+ *
+ * The stable step size factors for different orders are given by:
```

Comment:
Is there a reference for this table? If so, might be nice to cite it to provide a starting point for anyone seeking to learn more...

Reply:
I don't know, I extracted the numbers from the function in the cpp file :man_shrugging:

Reply:
Okay, I was just asking in case there was an easy reference to add, but if not then this is already a nice improvement and should go in as is :)

Reply:
If anyone knows of a reference I'm happy to add it, but I'm unfortunately not aware of one :) Though I agree it would be nice to have :)

Reply:
I don't have a reference handy.  Here's what the code says, though:
```c++
  for (size_t order = 1; order < 9; ++order) {
    const double step = TimeSteppers::AdamsBashforthN(order).stable_step();
    std::cerr << order << " | "
              << simplest_fraction_in_interval<Rational>(step - 1e-12,
                                                         step + 1e-12) << "\n";
  }
```
```
1 | 1/1
2 | 1/2
3 | 3/11
4 | 3/20
5 | 45/551
6 | 5/114
7 | 945/40633
8 | 945/77432
```